scpdt/tests/conftest.py

```diff
@@ -0,0 +1,10 @@
+from scpdt.impl import DTConfig
+
+
+config = DTConfig()
+config.local_resources = {'scpdt.tests.local_file_cases.local_files':
+                                  ['scpdt/tests/local_file.txt'],
+                            'scpdt.tests.local_file_cases.sio':
+                                  ['scpdt/tests/octave_a.mat']   
+                                  }
+
```

Comment:
For my education: what's the preferred / recommended location for the conftest.py file: in the `tests/`  folder or at the main package level? Numpy seems to use the latter: 
https://github.com/numpy/numpy/tree/main/numpy

Maybe @melissawm has a suggestion?

Reply:
https://docs.pytest.org/en/7.1.x/how-to/writing_plugins.html#plugin-discovery-order-at-tool-startup
This is what I can find about that. Should probably be in the root directory or a root test directory.
@melissawm any suggestions?